components/signup/signup_email/signup_email.tsx

```diff
@@ -23,7 +26,43 @@ import SiteNameAndDescription from 'components/common/site_name_and_description'
 
 import FormattedMarkdownMessage from 'components/formatted_markdown_message.jsx';
 
-export default class SignupEmail extends React.PureComponent {
+import {Actions} from 'components/signup/signup_email/index';
```

Comment:
Does Actions still need to be defined in the index file or could it be defined here and imported from the index?

Reply:
No I don't think so. I'll move it to this file to change where it's imported from